### DISCUSSION TITLE
Header/Freeze row Customisation Help

### DISCUSSION THREAD

**Discussion by ArnavWee** [glideapps/glide-data-grid#906] [Q&A]
Hi , really love the grid and my team is one step away from migrating to this. There is a big feature that I am not sure if we will be able to support.

I want do something like this, have a header like row which is sticky/frozen. One way I was thinking of doing this is was by utilising frozen rows but looking at the API i can see that those are only possible to have at the end of the table. The other options would be to be able to change the header box itself and add custom html for this ( I don't think this is possible maybe)?
 What do you think is the best way to do this
<img width="449" alt="Screenshot 2024-02-23 at 4 27 31 PM" src="https://github.com/glideapps/glide-data-grid/assets/108408282/1d9eacb6-f616-4b8e-90ab-283114d6291a">

**Comment by sduongCS** [comment]
bump
